piccolo_admin/endpoints.py

```diff
@@ -585,6 +594,11 @@ def __init__(
             app=StaticFiles(directory=os.path.join(ASSET_PATH, "js")),
         )
 
+        self.mount(
+            path="/icons",
+            app=StaticFiles(directory=os.path.join(ASSET_PATH, "icons")),
+        )
```

Comment:
I think this needs to be changed  - we should let the user have their custom image in any folder. If it has to be the `icons` folder it's problematic, as the user has to be put the file within the `piccolo_admin` folder - so every time they install a new version it gets overwritten.

Reply:
You're right. Can you change that and add a favicon and logo (whatever you think is best) because I have to go now.

Reply:
I leave this path for the default logo and favicon. User can set their custom logo and favicon server path in `create_admin` with `logo_path` and `favicon_path` arguments.

Reply:
Thanks. I'll try and merge this in tomorrow.

Reply:
Sorry, didn't get a chance to do this yet. I caught Covid recently, and my brain is running a bit slow.

Reply:
No worries.  I hope you get better.

Reply:
@dantownsend , all checks have passed. Whenever you feel better, you can merge this.

Reply:
I'm about half way through reviewing it - should be done soon.